packages/e2e-tests/src/specs/wordpress/tinymce.js

```diff
@@ -14,34 +14,28 @@
  * limitations under the License.
  */
 
-/**
- * WordPress dependencies
- */
-import {
-  activatePlugin,
-  deactivatePlugin,
-  visitAdminPage,
-} from '@wordpress/e2e-test-utils';
-
 /**
  * External dependencies
  */
 import percySnapshot from '@percy/puppeteer';
+import {
+  withPlugin,
+  withUser,
+  visitAdminPage,
+} from '@web-stories-wp/e2e-test-utils';
 
 describe('TinyMCE button', () => {
-  beforeAll(async () => {
-    await activatePlugin('classic-editor');
-  });
-
-  afterAll(async () => {
-    await deactivatePlugin('classic-editor');
-  });
+  withUser('admin', 'password');
+  withPlugin('classic-editor');
 
   it('should allow inserting shortcode via modal', async () => {
     await visitAdminPage('post-new.php');
 
     // Ensure we're in the visual editor.
     await expect(page).toClick('#content-tmce');
+    await expect(page).toMatchElement('.mce-tinymce.mce-container', {
```

Comment:
Neat!

Reply:
Added this to reduce flakiness in the test. It sometimes failed because `.mce-web-stories button` was missing, which can happen when the visual editor hasn't fully loaded yet.